CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changelog
 #########
 
+Version 1.11.1
```

Comment:
Shouldn't be an 1.12.0?

Reply:
Yeah 👍 I was thinking it was only a simple formality of removing the <3.0 and testing that it worked originally :D (how naïve)

Reply:
Done